include/fmt/core.h

```diff
@@ -223,7 +223,7 @@ class basic_buffer {
 
  protected:
   // Don't initialize ptr_ since it is not accessed to save a few cycles.
-  basic_buffer(std::size_t sz) FMT_NOEXCEPT: size_(sz), capacity_(sz) {}
+  basic_buffer(std::size_t sz) FMT_NOEXCEPT: ptr_(FMT_NULL), size_(sz), capacity_(sz) {}
```

Comment:
`ptr_` is not initialized intentionally here. This have measurable performance impact on some benchmarks.

Reply:
OK - as long as the header does not produce warnings when included in user-level code.